ReturnForm.html

```diff
@@ -0,0 +1,9 @@
+$def with (loan)
+
+$# "do_return" is a fake "title" for the book that can be used for stats
+$ return_url = loan['book'] + '/do_return/borrow'
+
+<form class="returnbook" action="$return_url" method="post">
+<input type="submit" class="returnb" value="$('Return book')" />
```

Comment:
```suggestion
<input type="submit" class="returnb" value="_$('Return book')" />
```

Missing underscore(_)